.github/workflows/release_gh_pages.yml

```
@@ -25,7 +25,9 @@ jobs:
           node-version: '16'
 
       - name: Install wine
-        run: sudo apt-get install -y wine-stable
+        run: |
+          sudo apt update
```

Maybe use apt-get instead, since apt does not have a stable cli interface
```suggestion
          sudo apt-get update
```